Infusionsoft/TokenStorageProvider.php

```diff
@@ -9,5 +9,4 @@
 interface Infusionsoft_TokenStorageProvider {
     public function saveTokens($appDomainName, $accessToken, $refreshToken, $expiresIn);
     public function getTokens($appDomainName);
-    public function getFirstAppName();
```

Comment:
Our implementation of the TokenStorageProvider only allows the correct app to authenticate, we never reference getFirstAppName and it did not seem appropriate to require it in TokenStorageProvider classes anymore